charts/aws-efs-csi-driver/templates/controller-deployment.yaml

```diff
@@ -32,29 +32,29 @@ spec:
       {{- end }}
       nodeSelector:
         kubernetes.io/os: linux
-        {{- with .Values.nodeSelector }}
+        {{- with .Values.controller.nodeSelector }}
         {{- . | toYaml | nindent 8 }}
         {{- end }}
       serviceAccountName: {{ include "aws-efs-csi-driver.serviceAccountName" . }}
       priorityClassName: system-cluster-critical
       tolerations:
         - operator: Exists
-        {{- with .Values.node.tolerations }}
+        {{- with .Values.controller.tolerations }}
         {{- . | toYaml | nindent 8 }}
         {{- end }}
       containers:
         - name: efs-plugin
           securityContext:
             privileged: true
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
```

Comment:
Can we move `pullPolicy` to their own fields under `node` and `controller`? Just to be consistent.

Reply:
that would be less consistent cuz it would break the pattern of image structs all having the same fields pullPolicy, tag , & repository. The top-level `image` is the driver image which should be the same for both node and controller.